nextcord/embeds.py

```diff
@@ -183,33 +170,34 @@ class Embed:
         "description",
     )
 
-    Empty: Final = EmptyEmbed
+    # backwards compatibility
+    Empty = EmptyEmbed
```

Comment:
Could this perhaps be some form of property with a `FutureWarning` (as default filters ignore `DeprecationWarning`), or `__getattr__` so users know it should not be used?

Reply:
Done!